8.19.3 release notes (#7045)
elastic/security-docs: docs/release-notes/8.19.asciidoc

[[release-notes-header-8.19.0]]
== 8.19

[discrete]
[[release-notes-8.19.3]]
=== 8.19.3

[discrete]
[[enhancements-8.19.3]]
==== Enhancements

> * Improves the reliability of {elastic-defend}'s connection to its kernel driver. This should reduce the instances of temporary `DEGRADED` policy statuses at boot due to `connect_kernel` failures.

Reviewer: ```suggestion
* Improves the reliability of {elastic-defend}'s connection to its kernel driver. This should reduce the instances of temporary `DEGRADED` policy statuses at boot due to `connect_kernel` failures.
* Enriches {elastic-defend} macOS network connect events with `network.direction`. Possible values are ingress and egress.
```

[discrete]
[[bug-fixes-8.19.3]]
==== Fixes

> * Prevents Session View from crashing by normalizing the `event.process.args` field ({kibana-pull}232462[#232462]).

Reviewer 2: ```suggestion
* Fixes a bug in Session View where `args` fields in `event.process` and nested objects had string values instead of array of strings as expected ({kibana-pull}232462[#232462]).
```

> * Fixes a bug in {elastic-defend} where Linux endpoints would report `process.executable` as a relative, instead of absolute, path.

Reviewer: ```suggestion
* Fixes a bug in {elastic-defend} where Linux endpoints would report `process.executable` as a relative, instead of absolute, path.
* Fixes a bug where {elastic-defend} Linux network events would fail to load if ipv6 is not supported by the system.
* Fixes a bug in {elastic-defend} where the fqdn feature flag was not being persisted across system/endpoint restarts
* Fix a race condition in {elastic-defend} that may occasionally result in corrupted process command lines on Windows.  When this occurs, `process.command_line`, `process.args_count` and `process.args` may be incorrect, leading to false positives.
* Fixes a bug in {elastic-defend} where Linux endpoints would report `process.executable` as a relative, instead of absolute, path.
```

Reviewer 3: `Fixes a bug in {elastic-defend} where Linux endpoints would report `process.executable` as a relative, instead of absolute, path.`. That is repeated in the suggestion.

[[release-notes-header-8.19.0]]
== 8.19

[discrete]
[[release-notes-8.19.3]]
=== 8.19.3

[discrete]
[[enhancements-8.19.3]]
==== Enhancements
* Improves the reliability of {elastic-defend}'s connection to its kernel driver. This should reduce the instances of temporary `DEGRADED` policy statuses at boot due to `connect_kernel` failures.
* Enriches {elastic-defend} macOS network connect events with `network.direction`. Possible values are `ingress` and `egress`.

[discrete]
[[bug-fixes-8.19.3]]
==== Fixes
* Fixes a bug in Session View where `args` fields in `event.process` and nested objects had string values instead of array of strings as expected ({kibana-pull}232462[#232462]).
* Fixes a bug in {elastic-defend} where Linux endpoints would report `process.executable` as a relative, instead of absolute, path.
* Fixes a bug where {elastic-defend} Linux network events would fail to load if IPv6 is not supported by the system.

> * Fixes a bug in {elastic-defend} where the `fqdn` feature flag was not being persisted across system/endpoint restarts.

Reviewer 4: Optional nit!

```suggestion
* Fixes a bug in {elastic-defend} where the `fqdn` feature flag was not being persisted across system or endpoint restarts.
```

> * Fixes a race condition in {elastic-defend} on Windows that occasionally resulted in corrupted process command lines. This could cause incorrect values for `process.command_line`, `process.args_count`, and `process.args`, leading to false positives.

Reviewer 4: ```suggestion
* Fixes a race condition in {elastic-defend} on Windows that occasionally resulted in corrupted process command lines. This could cause incorrect values for `process.command_line`, `process.args_count`, and `process.args`, leading to false positives.
* Hides case connectors in the create case workflow based on your license ({kibana-pull}232506[#232506]).
* Fixes inconsistencies in case activity statistics ({kibana-pull}231948[#231948]).
```

[[enhancements-8.19.3]]
==== Enhancements
* Improves the reliability of {elastic-defend}'s connection to its kernel driver. This should reduce the instances of temporary `DEGRADED` policy statuses at boot due to `connect_kernel` failures.
* Enriches {elastic-defend} macOS network connect events with `network.direction`. Possible values are ingress and egress.

[discrete]
[[bug-fixes-8.19.3]]
==== Fixes
* Fixes a bug in Session View where `args` fields in `event.process` and nested objects had string values instead of array of strings as expected ({kibana-pull}232462[#232462]).
* Fixes a bug in {elastic-defend} where Linux endpoints would report `process.executable` as a relative, instead of absolute, path.
* Fixes a bug where {elastic-defend} Linux network events would fail to load if ipv6 is not supported by the system.
* Fixes a bug in {elastic-defend} where the fqdn feature flag was not being persisted across system/endpoint restarts
* Fix a race condition in {elastic-defend} that may occasionally result in corrupted process command lines on Windows.  When this occurs, `process.command_line`, `process.args_count` and `process.args` may be incorrect, leading to false positives.

> * Fixes a bug in {elastic-defend} where Linux endpoints would report `process.executable` as a relative, instead of absolute, path.

Reviewer 3: ```suggestion
```
This is repeated on line 18

Author: Thanks for flagging @nicholasberlin, I'm on it :)
